index.html

```diff
@@ -245,7 +245,46 @@ <h4 class="bolda" >About me</h4>
           </main>
         </section>
     
-    
+    <footer class="footer">
+      <h2 class="footer-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in <span class="touch">touch.</span></h2>
+      
+      <form  class="formx" action="https://formspree.io/f/xayagqrn" method="post">
+        <ul class="formul">
+        <li class="for"><label for="fullName"><input type="text" name="full-name" placeholder="  Full name" maxlength="30" id="fullName" required>
+        </label>
+      </li>
+      <li class="for">
+        <label for="email">
+          <input type="email" name="email" id="email" placeholder="  Email address" required>
+        </label>
+      </li>
+      <li class="textare">
+        <label for="message"><textarea name="message" id="message" cols="30" rows="6" maxlength="500" required>Enter text here</textarea></label>
```

Comment:
- [ ] Let's add a `placeholder` attribute to the `textarea` tag. You can remove the text inside the tag. This is not only a standard, but also provides a better user experience since the user doesn't have to delete manually the placeholder text.